addons/acre/functions/fnc_copyLeaderRadioChildrenActions.sqf

```diff
@@ -0,0 +1,61 @@
+#include "script_component.hpp"
+/*
+ * Author: veteran29, ACRE2Team
+ * Get child actios to copy radio settings from leader, based on ACRE2 fnc_radioListChildrenActions function.
+ *
+ * Arguments:
+ * 0: Objects <ARRAY>
+ * 1: All <BOOL>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [[bob, ted], false] call afm_main_fnc_example
```

Comment:
```suggestion
 * Arguments:
 * 0: Unit to copy radio settings from it's leader <OBJECT>
 *
 * Return Value:
 * List of actions <ARRAY>
 *
 * Example:
 * [player] call afm_acre_fnc_copyLeaderRadioChildrenActions
```

Return value could require more explanation.

Reply:
It's an internal function for ACE interaction nodes but I'll try to write something more descriptive.